### ISSUE TITLE
About test video's framerate

### ISSUE THREAD

**Issue by liuping0804** [nxsEdson/CVD-Physiological-Measurement#6]
I recently implemented the training part of your model with the VIPL dataset. Among them, the training loss can indeed reach below 1, but the test loss is relatively large when tested with the VIPL_V2 data set. I have the following questions:
         1) In addition to the frame rate used in the SNR calculation part of the model, what is the relationship between the heart rate value output of the model and the frame rate of the video;
         2) The output of the model is normalized by the frame rate. The heart rate value needs to be de-normalized during the test, and the error will be very large at this time. Since the training frame rate is variable, can it be considered that the video frame rate does not need to be fixed for testing?

**Comment by yuhang1070** [comment]
Did you get the same result as the paper in vipl-hr dataset?

**Comment by yuhang1070** [comment]
> I recently implemented the training part of your model with the VIPL dataset. Among them, the training loss can indeed reach below 1, but the test loss is relatively large when tested with the VIPL_V2 data set. I have the following questions:
> 1) In addition to the frame rate used in the SNR calculation part of the model, what is the relationship between the heart rate value output of the model and the frame rate of the video;
> 2) The output of the model is normalized by the frame rate. The heart rate value needs to be de-normalized during the test, and the error will be very large at this time. Since the training frame rate is variable, can it be considered that the video frame rate does not need to be fixed for testing?

Did you get the same result as the paper in vipl-hr dataset?

**Comment by yuhang1070** [comment]
> I recently implemented the training part of your model with the VIPL dataset. Among them, the training loss can indeed reach below 1, but the test loss is relatively large when tested with the VIPL_V2 data set. I have the following questions:
> 1) In addition to the frame rate used in the SNR calculation part of the model, what is the relationship between the heart rate value output of the model and the frame rate of the video;
> 2) The output of the model is normalized by the frame rate. The heart rate value needs to be de-normalized during the test, and the error will be very large at this time. Since the training frame rate is variable, can it be considered that the video frame rate does not need to be fixed for testing?

Now epoch = 61
------------------------------------------------------------
Train
mae: 1.1605256233052055, rmse: 1.6857475755523663, r: 0.9931153538313953
------------------------------------------------------------
Test epoch: test_loss: 2577.2662741839886
mae: 23.061266541701347, rmse: 29.167665694023405, r: 0.007970992640142758

I get something wrong. Did I overlook any key points?

**Comment by shayan113** [comment]
> > I recently implemented the training part of your model with the VIPL dataset. Among them, the training loss can indeed reach below 1, but the test loss is relatively large when tested with the VIPL_V2 data set. I have the following questions:
> > 
> > 1. In addition to the frame rate used in the SNR calculation part of the model, what is the relationship between the heart rate value output of the model and the frame rate of the video;
> > 2. The output of the model is normalized by the frame rate. The heart rate value needs to be de-normalized during the test, and the error will be very large at this time. Since the training frame rate is variable, can it be considered that the video frame rate does not need to be fixed for testing?
> 
> ## Now epoch = 61
> ## Train
> mae: 1.1605256233052055, rmse: 1.6857475755523663, r: 0.9931153538313953
> Test epoch: test_loss: 2577.2662741839886
> mae: 23.061266541701347, rmse: 29.167665694023405, r: 0.007970992640142758
> 
> I get something wrong. Did I overlook any key points?

Can you share your dataloader，the training and testing code?

**Comment by TarrySun** [comment]
> > I recently implemented the training part of your model with the VIPL dataset. Among them, the training loss can indeed reach below 1, but the test loss is relatively large when tested with the VIPL_V2 data set. I have the following questions:
> > 
> > 1. In addition to the frame rate used in the SNR calculation part of the model, what is the relationship between the heart rate value output of the model and the frame rate of the video;
> > 2. The output of the model is normalized by the frame rate. The heart rate value needs to be de-normalized during the test, and the error will be very large at this time. Since the training frame rate is variable, can it be considered that the video frame rate does not need to be fixed for testing?
> 
> ## Now epoch = 61
> ## Train
> mae: 1.1605256233052055, rmse: 1.6857475755523663, r: 0.9931153538313953
> Test epoch: test_loss: 2577.2662741839886
> mae: 23.061266541701347, rmse: 29.167665694023405, r: 0.007970992640142758
> 
> I get something wrong. Did I overlook any key points?

I was also unable to achieve the results in the paper, the Pearson correlation tested was around 0.6 and overfitting occurs after 40 epochs, and I used the VIPL-HR dataset, what about you now?

**Comment by shanchao0906** [comment]
> I recently implemented the training part of your model with the VIPL dataset. Among them, the training loss can indeed reach below 1, but the test loss is relatively large when tested with the VIPL_V2 data set. I have the following questions: 1) In addition to the frame rate used in the SNR calculation part of the model, what is the relationship between the heart rate value output of the model and the frame rate of the video; 2) The output of the model is normalized by the frame rate. The heart rate value needs to be de-normalized during the test, and the error will be very large at this time. Since the training frame rate is variable, can it be considered that the video frame rate does not need to be fixed for testing?

Hello, I would like to know the specific meaning of data set content in VIPL-HR. 1. Is file GT-HR.csv heart rate per second?2. What is time. txt? 3.  Wave.CSV is BVP data？4.Is the video time aligned with the contact device, and is the sampling frequency the same?

**Comment by whr1241** [comment]
> I recently implemented the training part of your model with the VIPL dataset. Among them, the training loss can indeed reach below 1, but the test loss is relatively large when tested with the VIPL_V2 data set. I have the following questions: 1) In addition to the frame rate used in the SNR calculation part of the model, what is the relationship between the heart rate value output of the model and the frame rate of the video; 2) The output of the model is normalized by the frame rate. The heart rate value needs to be de-normalized during the test, and the error will be very large at this time. Since the training frame rate is variable, can it be considered that the video frame rate does not need to be fixed for testing?

I download the VIPL-HR-V2 dataset，but I don't know the password.Could you tell me?
